Source/Tests/TrackableEntities.EF.5.Tests/FamilyDbContextTests.cs

```diff
@@ -215,5 +215,21 @@ public void Apply_Changes_Should_Mark_Grandchild_Deleted()
             Assert.Equal(EntityState.Deleted, context.Entry(child).State);
             Assert.Equal(EntityState.Deleted, context.Entry(grandchild).State);
         }
+
+        [Fact]
+        public void Apply_Changes_Should_Mark_ComplexType_Modified()
+        {
+            // Arrange
+            var context = TestsHelper.CreateFamilyDbContext(CreateFamilyDbOptions);
+            var parent = new MockFamily().Parent;              
+            parent.Address.StreetName = "123 Lee Ave.";
+            parent.Address.ModifiedProperties = new[] { nameof(Address.StreetName) };
```

Comment:
So change the code in `DbContextExtensions` so that this test fails.  Then go back and change the test to add:

```csharp
parent.Address.TrackingState = TrackingState.Modified
```

Then the test will pass.